taiga_halo2/examples/sudoku/main.rs

```diff
@@ -53,6 +53,7 @@ fn main() {
     pub_instance_vec.append(&mut vec_puzzle);
     assert_eq!(
         MockProver::run(13, &circuit, vec![pub_instance_vec.clone()])
+        // TODO:        ^^ should this '13' be 'K'?
```

Comment:
Yes

Reply:
What is K btw? Was just looking at the sudoku code a few days ago and couldn't figure it out

Reply:
The size parameter of the circuit (which has 2^k rows).

The more complex the circuit it, the larger this needs to be and the slower it runs